std/bitmanip.d

```diff
@@ -309,51 +316,6 @@ one bitfield with an empty name.
     }
 }
 
-/**
-Creates a bitfield pack of eight bits, which fit in
-one `ubyte`. The bitfields are allocated starting from the
-least significant bit, i.e. x occupies the two least significant bits
-of the bitfields storage.
-*/
-@safe unittest
-{
-    struct A
-    {
-        int a;
-        mixin(bitfields!(
-            uint, "x",    2,
-            int,  "y",    3,
-            uint, "z",    2,
-            bool, "flag", 1));
-    }
-    A obj;
-    obj.x = 2;
-    obj.z = obj.x;
-
-    assert(obj.x == 2);
-    assert(obj.y == 0);
-    assert(obj.z == 2);
-    assert(obj.flag == false);
-}
-
-/// Add empty fields for padding to have a total bit length of 8, 16, 32, or 64
-@safe unittest
-{
-    struct A
```

Comment:
...this `A` however is different from the one above. Please leave this one as is.

Reply:
Where's the difference? I can't see any. I even tried the diff utility...

Reply:
```diff
-             uint, "x",    2,		
-             int,  "y",    3,		
-             uint, "z",    2,		
-             bool, "flag", 1));
+            bool, "flag1",    1,		
+             bool, "flag2",    1,		
+             uint, "",         6));
```

Reply:
Ah no, this are not the two I compared: See lines 289 and 342 in the original file. They are both this 1, 1, 6 pattern. Just the comment is different.

Reply:
Ah, my bad.